### ISSUE TITLE
Option to hide tile loading errors after N seconds

### ISSUE THREAD

**Issue by hrvoj3e** [tify-iiif-viewer/tify#148]
Could you add an option to hide tile loading errors after N seconds?
Or an option to stop showing them to users and log to console only.

**Comment by t11r** [comment]
Good idea. You don’t happen to have a IIIF manifest URL which reliably produces tile loading errors?

**Comment by hrvoj3e** [comment]
No. Sorry. 
But use any manifest and manually edit some identifiers for resources. This will have a similar end result...

**Comment by t11r** [comment]
Image loading errors are now removed on page change. We decided against a timer for accessibility reasons.
